### ISSUE TITLE
Write a Sanity Check against duplicate ScaleExponents!

### ISSUE THREAD

**Issue by Lisias** [net-lisias-ksp/TweakScale#156]
This can play havoc with savegames!

**Comment by sswelm** [comment]
well in the past I several changed my own ScaleExponents, and they never generated any real issues, something just became different at statup.

**Comment by Lisias** [comment]
In the past, whatever was placed on the craft file and on the savegame had precedence over the prefab.

But nowadays, I'm guessing since KSP 1.5 or 1.6, KSP is shoving current prefab data on the craft and savegames on loading.

You could edit the parts atributes on the craft file on KSP 1.4 (I used to do that to test things faster), but now the values are reset to the prefab when loading the craft, and my stunt ceased to work.

So, if you change a ScaleExponent, **every single part** on the loading craft and savegames will be "upgraded" to this new Exponent on load. But the user was using the previous one, so any resource planning, weigh balancing, and everything else that depended on that Exponent will be completely different from what the user is expecting.

**Comment by sswelm** [comment]
Yes, this is one of the main reasons why I maintain support for Tweakscale myself, I can change everything as I see fit. For me, this is part of the package deal. I understand you need to be more careful

**Comment by Lisias** [comment]
I just remembered that I can't implement this without getting rid of the deprecating patches first.

So I'm postponing it to 2.5 release.
